tdrs-backend/tdpservice/search_indexes/parsers/tanf_parser.py

```diff
@@ -95,3 +98,14 @@ def parse(datafile):
         else:
             logger.warn("Parsing for type %s not yet implemented", record_type)
             continue
+
+def validate(family_case_schema, t1):
+    """Validate the datafile."""
+    errors = []
+    for field in family_case_schema:
+        for validator in field['validators']:
+            if validator(t1) is False:
+                errors.append({
+                    'field': field,
+                    'validator': validator,
```

Comment:
Not sure how to formulate the errors. These objects should have all the pertinent information.

Reply:
#1121 will introduce the object to be used here. Whatever that object ends up being, it looks like `errors` will be a list of those instances. 

> These objects should have all the pertinent information.

Mhm, agreed. 1121 lists out a bunch of wishlist fields like `line number`, etc. We might need to pass more arguments into this function to have that information available.